### PR TITLE
Delete new_remote.txt

### PR DESCRIPTION
**Pull request by senthilkgh**
view

### CODE REVIEW

**Review by tempsen — APPROVED** [review]
reviewed and approved

**Review by tempsen — APPROVED** [review]
checked viewed approved

**Review by tempsen — APPROVED** [review]

**Review by tempsen — APPROVED** [review]
ee

**Review by tempsen — APPROVED** [review]

File: new_remote.txt
Comment (tempsen) [inline new_remote.txt:1]: fsdgsfg

**Review by tempsen — APPROVED** [review]
rerwter

**Comment by senthilkgh** [comment]
sdfsdf

**Pull request closed without merging**